docs/topics/native/native-memory-manager.md

```diff
@@ -62,10 +66,27 @@ To track GC-related pauses in your app:
 
    Here, each blue blob on the lowest graph represents a separate signpost event, which is a GC pause.
 
+### Optimize GC performance
+
+To improve GC performance, you can try a couple of options aimed at decreasing the GC pause time:  
+
+* For the marking phase, you can use concurrent marking. It allows to run the marking phase of the garbage collection
+  simultaneously with application threads.
+
+  The feature is currently [Experimental](components-stability.md#stability-levels-explained). To enable it, set the
+  following option in your `gradle.properties` file:
+  
+  ```properties
+  kotlin.native.binary.gc=cms
+  ```
+
+* For the next phase of handling weak references, you can enable concurrent processing by using the
+  `-Xbinary=concurrentWeakSweep=true` compiler option.
```

Comment:
The `concurrentWeakSweep` does not make sense with the CMS GC (i.e. always enabled with CMS), and is the default behavior with the Parallel Mark (default) GC. 
I think, we should remove this item.

Reply:
OK, then since it's enabled by default, I mentioned it in the general description of the GC. The only question here is do we want now to add info on how to disable `concurrentWeakSweep`?